### PR TITLE
uglify css for high contrast mode

### PR DESCRIPTION
**Pull request by agessler**
Fix for https://trello.com/c/n5l4xLwL/93-high-contrast-mode

### CODE REVIEW

**Comment by apepper** [comment]
To see this in action one can select "Increase contrast" in the Mac OS settings:

high contrast mode

File: src/assets/stylesheets/index.scss [hunk 1]
```diff
@@ -1756,3 +1756,40 @@ header.portal-only section:after {
...
+  }
+  .Toastify__toast,
+  .navbar-toggler.btn,
+  .form-control,
+  .form-select,
+  .card,
+  .editor-img,
+  .box-comment .avatar img,
+  .box-attachment {
+    border: 1px solid #777;
```
Comment (apepper) [inline src/assets/stylesheets/index.scss:1778]: I found one place, where the border is not (yet) working nicely: In the `DataMessageWidget` - the arrow to the avatar should also have a border:

Screenshot 2024-03-01 at 12 12 08

**Review by apepper — CHANGES_REQUESTED** [review]
FEEDBACK ([a164ded](https://github.com/Scrivito/scrivito-portal-app/pull/318/commits/a164ded687330287571b870df9396c221cd3afe8))

Very nice 🚀!

I only found one smaller issue (see comments). I also did not yet was able to try it out with the real beamer in question.

**Comment by agessler** [comment]
@apepper Fixed.. I saw this issue.. I was lazy yesterday.

**Review by apepper — APPROVED** [review]
LGTM ([990ba8a](https://github.com/Scrivito/scrivito-portal-app/pull/318/commits/990ba8a971e1f1ac2e5b16aa95af4dade999f1ba))

Nice 🚀!

**Pull request merged**